src/stories/src/components/split-view/split-view.mdx

```diff
@@ -0,0 +1,265 @@
+import {
+  Default as SplitViewDemo,
+  LeadingPanel as LeadingDemo,
+  MultiplePanelsSplit as MultipleSplitDemo,
+  MultiplePanelsStacked as MultipleStackedDemo
+} from './split-view.stories.tsx';
+import { SplitViewDefaultCodeHtml } from './code/split-view-default.ts';
+import { SplitViewLeadingCodeHtml } from './code/split-view-leading.ts';
+import { SplitViewVerticalCodeHtml } from './code/split-view-vertical.ts';
+import { SplitViewMultipleSplitCodeHtml } from './code/split-view-multiple-split.ts';
+import { SplitViewMultipleStackedCodeHtml } from './code/split-view-multiple-stacked.ts';
+
+<PageSection>
+
+# Split view
+
+The split view component presents `<forge-split-view-panel>` elements side-by-side as resizable panels.
```

Comment:
Let's build on this overview and some more context around what it is used for, why it might be used, and when it might be used. We can add some ground rules for the component at a high level here too.